modules/fedimint-lnv2-server/src/lib.rs

```diff
@@ -370,6 +372,11 @@ impl ServerModule for Lightning {
 
                 ensure!(current_vote < vote, "Unix time vote is redundant");
 
+                ensure!(
+                    current_vote + MIN_UNIX_TIME_VOTE_INCREMENT_SECS < vote,
+                    "Unit time vote is not incremented enough"
+                );
```

Comment:
This is a consensus breaking change. Can we just propose less often instead? I think that's what @m1sterc001guy already mentioned.

Reply:
Although, it's LNv2 so we might not care abut consensus breaks yet, I just got freaked out a bit by seeing a consensus change being proposed :sweat_smile:

Reply:
I think we can still break, and I don't think "proposing things less often" does not fit well our proposal model.

Reply:
> does not fit well our proposal model

I know, we don't do that anywhere, I'd still like to though :sweat_smile: I think one of the reasons for the "always submit" approach was that CIs were meant to only be sent infrequently (every 30s or so) and that future *could* be here once LNv1 is gone (only module that still relies on low latency CIs).